library-model/test-library-model-generator/src/main/java/com/uber/nullaway/libmodel/provider/TestProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.uber.nullaway.libmodel.provider;
+
+import com.google.auto.service.AutoService;
+import com.uber.nullaway.jarinfer.JarInferStubxProvider;
+import java.util.Collections;
+import java.util.List;
+
+@AutoService(JarInferStubxProvider.class)
```

Comment:
I guess another thing to rename (now or in the future), to `StubxProvider`?

Reply:
Agreed, but not for this PR